doc/api.md

```diff
@@ -338,6 +339,21 @@ Use this method to send video files, Telegram clients support mp4 videos (other
 | video | <code>String</code> &#124; <code>stream.Stream</code> &#124; <code>Buffer</code> | A file path or Stream. Can also be a `file_id` previously uploaded. |
 | [options] | <code>Object</code> | Additional Telegram query options |
 
+<a name="TelegramBot+sendVideoNote"></a>
+
+### telegramBot.sendVideoNote(chatId, videoNote, [options]) ⇒ <code>Promise</code>
+Use this method to send rounded square videos of upto 1 minute long.
+
+**Kind**: instance method of <code>[TelegramBot](#TelegramBot)</code>  
+**See**: https://core.telegram.org/bots/api#sendvideonote  
+**Note**: The length parameter is optional, but **required** untill Telegram patches it.
```

Comment:
Please move this into the source code. Otherwise, it will be overwritten once the docs are re-generated.

To be more precise: *The `length` parameter is actually **optional**. However, the API (at time of writing) requires you to **always** provide it  until it is fixed.*